src/routes/edit/share-options/exports/frameworks/angular/fragment-v10.ts

```diff
@@ -75,6 +76,22 @@ const getComponentCode = (fragment: any, fragments: any[]) => {
 	return componentCode;
 };
 
+const hasPlaceholder = (json: any) => {
```

Comment:
Is there a reason this isn't `needsPlaceholder` as well?

Reply:
No specific reason